src/components/global/Task/Subtask.jsx

```diff
@@ -10,10 +10,15 @@ function Subtask(props) {
   return (
     <div className="subtask-container">
       <div
-        className={"subtask-checkbox" + (isChecked ? "--checked" : "")}
+        className={`subtask-checkbox${  isChecked ? "--checked" : ""}`}
         onClick={handleCheckBoxClick}
+        onKeyDown={handleCheckBoxClick}
+        role="checkbox"
```

Comment:
@thakurtushar02 A confirmation from you would be good since you implemented this component. Same applies to `src/components/global/IconButton.jsx` and `src/components/global/Task/Task.jsx`

Reply:
I also implemented this. Why do we need these extra fields?

Reply:
It's from eslint(jsx-a11y/no-static-element-interactions) rule that: `Static HTML elements with event handlers require a role.`
Documentations can be found here: [Jsx-eslint](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md)